docker-compose.yml

```diff
@@ -1,11 +1,51 @@
 version: "3.9"
 services:
-  wordpress:
-    # Here you put configuration of continer for Load balancer that would handle traffic to wordpress. Ports of webserver should be exposed to lacalhost
-    #...
-  wordpress:
-    # Here you put configuration of continer for wordpress that would use image that you build.
-    # ...
   database:
-    # Here you put configuration of continer for MySQL that would use any mysql image you like
-    #...
+    image: mysql:8
+    ports:
+    - "3306:3306"
+    restart: unless-stopped
+    env_file: .env
+    volumes:
+      - dbdata:/var/lib/mysql
+    command: '--default-authentication-plugin=mysql_native_password'
+ 
+
+  wordpress:
+    build: .
+    ports:
+      - "6060:80"
+    restart: always
+    volumes:
+      - ./wp-content/:/var/www/app/wordpress/wp-content
+    env_file: .env
+    environment:
+      - SLEEP_TIME=18
+    depends_on:
+      - database
+
+  wordpress2:
+    build: .
+    ports:
+      - "6062:80"
+    restart: always
+    volumes:
+      - ./wp-content/:/var/www/app/wordpress/wp-content
+    env_file: .env
+    environment:
+      - SLEEP_TIME=28
```

Comment:
why do you need sleep time? have you tried healthcheck ?

Reply:
I evaluated the idea, but it seems that ```depends_on``` and healthchecks don't work together anymore, i.e., services start without waiting for other services' startup.

Reply:
Hm. You are right. we cannot combine two options here. Your solution is fine.